extensions/resource-deployment/src/ui/deployAzureSQLDBWizard/deployAzureSQLDBWizard.ts

```diff
@@ -0,0 +1,153 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as azdata from 'azdata';
+import * as vscode from 'vscode';
+import * as constants from './constants';
+import { INotebookService } from '../../services/notebookService';
+import { IToolsService } from '../../services/toolsService';
+import { WizardBase } from '../wizardBase';
+import { WizardPageBase } from '../wizardPageBase';
+import { DeployAzureSQLDBWizardModel } from './deployAzureSQLDBWizardModel';
+import { AzureSQLDBWizardInfo } from '../../interfaces';
+import { AzureSettingsPage } from './pages/azureSettingsPage';
+import { DatabaseSettingsPage } from './pages/databaseSettingsPage';
+import axios, { AxiosRequestConfig } from 'axios';
+import { AzureSQLDBSummaryPage } from './pages/summaryPage';
+
+export class DeployAzureSQLDBWizard extends WizardBase<DeployAzureSQLDBWizard, WizardPageBase<DeployAzureSQLDBWizard>, DeployAzureSQLDBWizardModel> {
+
+	public get notebookService(): INotebookService {
+		return this._notebookService;
+	}
+
+	public get toolService(): IToolsService {
+		return this._toolsService;
+	}
+
+	constructor(private wizardInfo: AzureSQLDBWizardInfo, private _notebookService: INotebookService, private _toolsService: IToolsService) {
```

Comment:
nit: Put the constructor as the first thing in the class before the getters

Reply:
This is following @aasimkhan30 's code to be exact, along with the other lines, but I'll change it once I notify Aasim about it.